index.js

```diff
@@ -318,7 +318,9 @@ Tree.prototype._join = function (data) {
     }
   }
 
+  this.overflowClassed(data.length)
```

Comment:
In SB3 we've traditionally called methods like this `resize`.  What would you think of that as a name?